File: algorithms/strings/remove_blanks.py
def remove_blanks(myString):
    ln = len(myString)
    newString = ""
    for i in range(ln):
        if myString[i] != " ":
            newString += myString[i]
    return newString


myString = " Pl   ayTha tF u nkyM usi c  ";
print(remove_blanks(myString))

File: algorithms/strings/remove_blanks.rb
def remove_blanks(myString)
  len = myString.length
  newString = String.new()
  # for i in 0..len
  (0..len).each do |i|
    if myString[i] != " "
      newString << myString[i].to_s
    end
  end
  return newString;
end

myString = " Pl   ayTha tF u nkyM usi c  ";
puts remove_blanks(myString)
